### ISSUE TITLE
Updates to Mobility Bond Review (51093) on UC folder (SR) (Ready for UAT on PROD)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#16419]
### Updates to Mobility Bond Review (51093) on UC folder:

**Acceptance Criteria** 
- The AC provided applies only to UC folders 
- Update the name of the process Mobility Bond Review (51093) to CDS Review
- Under the Review section of Folder infos, update the info field Mobility Bond Review to CDS Review
- Default the CDS Review info field to Yes
- Update the merge document Mobility Bond Review Comments to CDS Review Comments
- Update Mobility Bond Reviewer (76707) info field to CDS Reviewer

**Comment by TracyLinder** [comment]
Attempt comment didn't merge into review doc
Approved with conditions not pulling in
Deborah F. is making updates to all the MCR's in the SP folder, which includes the UC template `Not good; need to stop`

**Comment by TracyLinder** [comment]
Pending testing results from G. Pepper

**Comment by TracyLinder** [comment]
@rajeshlaunch can you please touch base with Jenice and Gregory Pepper to get a status, and do they need help?

**Comment by TracyLinder** [comment]
**Update**

- Pepper has finished reviewing now CTM is working on the updates.

**Comment by vschandramouli** [comment]
Assigning Raj for visibility

**Comment by rajeshlaunch** [comment]
@vschandramouli  Working with Jenice to get more update on this

**Comment by rajeshlaunch** [comment]
@Nadin-Nader Is there any SN ticket that was created for CTM to work on this? When I checked with Jenice, she was not aware of such work and she may need assistance with an SN ticket # or something. Could you please help me with this?

**Comment by Nadin-Nader** [comment]
@rajeshlaunch i found a Mobility Bond Incident that Greg P.  created but it has to do with the Mobility Bond Review Comments on the Master Comment report. It is INC0390389 or it could be this one INC0411447

Other than that he did not have anything open for Mobility Bond Review

**Comment by rajeshlaunch** [comment]
Thank you @Nadin-Nader . Jenice is out today, I will check with her on Monday

**Comment by rajeshlaunch** [comment]
@Nadin-Nader sent the details to Jenice and waiting for her response.. I will keep you posted

**Comment by TracyLinder** [comment]
@vschandramouli as discussed please gather Pepper, Jaclyn, and Izzy to look at what's in the test system today, if it's good for PROD, and if not - capture the items that need fixing.

**Comment by vschandramouli** [comment]
While testing MCR, Greg P noticed it was pulling in assigned user instead of attempt user. and this was happening for some processes and not others. This needed to be addressed. Greg had already communicated with Hari. Vish has requested information on that conversation. Vish will talk to Tracy about scope creep on this card.

**Comment by vschandramouli** [comment]
Updated AC, moving to Backlog.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader - Analysis and Development is in progress

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader - This will impact SCP & LM Folders as this process is been used by UC, LM & SCP. I believe other teams are aware of the impact and I can continue with the changes. Pls Confirm. Thank you!

**Comment by TracyLinder** [comment]
@Sreelatha-BlazeKannan confirmed you may continue

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan the AC only states to do the name of the Review but doesnt say to update the reviewer. You will have to update the name of the review in the info just not the content

**Comment by vschandramouli** [comment]
@Sreelatha-BlazeKannan @Nadin-Nader Good catch. Please make that update as well. I updated the AC to match.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader - Submitted DEV DP Created servicenow task id: SCTASK0509800 .

Commit Details – 
DEV Staging - 8245bb7edb8eb3110aaf099981cba902d51a70bf

ATD Commit – 7ee8a487c5a4e0ac42590a51539fc8b5b3f0ebb6

VALIDINFO
DEFAULTINFO
VALIDPROCESS

Pull Request Link:

https://github.austintexas.gov/AMANDADevOps/Amandav7_TestingPOC/pull/1715

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan this has been sent for the manual deployment to CTM Support to be done under TASK **SCTASK0509854**

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader - Promoted to DEV; verified and tested in DEV as looks good, submitted TEST DP.

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan  This has been sent to the DevOps CM Team and to the CTM Support team to be deployed to TEST

DevOps Deployment: **SCTASK0510102**
Manual Deployment: **SCTASK0510494**

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader - Promoted to TEST and tested in TEST; all looks good. Ready for TEST UAT.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  Sree is already working on it, and as I still was seeing it on my board unassigned myself.
CC: @Sreelatha-BlazeKannan 
Thanks.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader - Ready for TEST UAT; Waiting for approval

**Comment by BondRuss** [comment]
@Nadin-Nader @Sreelatha-BlazeKannan Jac approved UAT in Test.

**Comment by BondRuss** [comment]
@Sreelatha-BlazeKannan Jac is going to do some UAT and I'll keep you posted. 
cc: @Nadin-Nader

**Comment by BondRuss** [comment]
@Sreelatha-BlazeKannan Jac's feedback, "CDS Review just needs to default to YES. Everything else looks fine."
cc: @Nadin-Nader @KyleErich
